game.conf

```diff
@@ -1,3 +1,3 @@
-name = Minetest Game
-author = Minetest
-description = A basic exploration, mining, crafting and building sandbox game with no NPCs, monsters or animals. Minetest Game is usually used with mods added and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
+name        = Minetest Game
+author      = Minetest
+description = A basic exploration, mining, crafting, and building, sandbox game with no NPCs, monsters, or animals. Minetest Game is usually used with mods added, and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
```

Comment:
A comma before `and` or `or`? I'm not a native speaker but that doesn't look right to me.

Reply:
This is called to as ["Serial comma"](https://en.wikipedia.org/wiki/Serial_comma). Both are correct in English.

Reply:
"foo, bar and baz" = `(foo and (bar and baz))`
"foo, bar, and baz" = `(foo and bar and baz)`
_Aber ich bin nur deutschsprechend Ausländer._ &nbsp; b^_~